MAUI/Segmented-Control/migration.md

```diff
@@ -0,0 +1,603 @@
+        };
+    }
+
+    public List<SfSegmentItem> ItemsCollection
+    {
+        get { return itemsCollection; }
+        set { itemsCollection = value; }
+    }
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+<table>
+<tr>
+<th>Xamarin SfSegmented control</th>
+<th>.NET MAUI SfSegmented control</th>
+<th>Description</th></tr>
+
+<tr>
+<td>{{'[FontColor](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.Buttons.SfSegmentedControl.html#Syncfusion_XForms_Buttons_SfSegmentedControl_FontColor)'| markdownify }}</td>
+<td>TextColor</td>
+<td>Gets or sets the text color for the SfSegmentedControl.</td>
+</tr>
+
+<tr>
+<td>{{'[FontSize](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.Buttons.SfSegmentedControl.html#Syncfusion_XForms_Buttons_SfSegmentedControl_FontSize)'| markdownify }}</td>
+<td>FontSize</td>
+<td>Gets or sets the double value that represents the font size of the SfSegmentedControl.</td>
+</tr>
+
+<tr>
+<td>{{'[FontFamily](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.Buttons.SfSegmentedControl.html#Syncfusion_XForms_Buttons_SfSegmentedControl_FontFamily)'| markdownify }}</td>
+<td>FontFamily</td>
+<td>Gets or sets the string, that represents font family of the SfSegmentedControl.</td>
+</tr>
+
+<tr>
+<td>{{'[FontAttributes](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.Buttons.SfSegmentedControl.html#Syncfusion_XForms_Buttons_SfSegmentedControl_FontAttributes)'| markdownify }}</td>
+<td>FontAttributes</td>
+<td>Gets or sets the FontAttributes of the SfSegmentedControl.</td>
+</tr>
+
+</table>
+
+## Enum
+
+<table>
+<tr>
+<th>Xamarin SfSegmented control</th>
+<th>.NET MAUI SfSegmented control</th>
+<th>Description</th></tr>
+
+<tr>
+<td>{{'[SelectionIndicatorPosition](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.Buttons.SelectionIndicatorPosition.html)'| markdownify }}</td>
+<td>SelectionIndicatorPlacement</td>
+<td>Describes the possible values for the position of the selection strip in SfSegmentedControl View.</td>
+</tr>
+
+</table>
+
+## Event
+
+<table>
+<tr>
+<th>Xamarin SfSegmented control</th>
+<th>.NET MAUI SfSegmented control</th>
+<th>Description</th></tr>
+
+<tr>
+<td>{{'[SelectionChanged](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.Buttons.SfSegmentedControl.html#Syncfusion_XForms_Buttons_SfSegmentedControl_SelectionChanged)'| markdownify }}</td>
+<td>SelectionChanged</td>
+<td>Occurs when the selection within the segment item is changed.</td>
+</tr>
+
+</table>
+
+## Methods
+
+<table>
+<tr>
+<th>Xamarin SfSegmented control</th>
+<th>.NET MAUI SfSegmented control</th>
+<th>Description</th></tr>
+
+<tr>
+<td>{{'[ScrollTo](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.Buttons.SfSegmentedControl.html#Syncfusion_XForms_Buttons_SfSegmentedControl_ScrollTo_System_Int32_Syncfusion_XForms_Buttons_ScrollToPosition_)'| markdownify }}</td>
+<td>ScrollTo</td>
+<td>Method to scrolls the scroll viewer to the specified index.</td>
```

Comment:
<td>Method to scroll the scroll viewer to the specified index.</td>

Reply:
content changed